.docker/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER Colin Fleming <c3flemin@gmail.com>
 # configure environment variable
 # note: move this to three ARG commands when CircleCI updates their docker
 ENV DCAF_DIR=/usr/src/app \
-    BUILD_DEPENDENCIES="build-base libxml2-dev libxslt-dev linux-headers bash openssh fontconfig fontconfig-dev curl" \
+    BUILD_DEPENDENCIES="build-base libxml2-dev libxslt-dev linux-headers bash openssh fontconfig fontconfig-dev curl ruby-dev alpine-sdk" \
```

Comment:
can you help me understand what's going on with these changes? I would have thought we'd have gotten ruby-dev and alpine-sdk from the base image, though I admittedly don't know that much about what alpine does/comes with!